### PR TITLE
[PAY-872][PAY-873][PAY-874] More premium content QA fixes

### PR DESCRIPTION
**Pull request by sddioulde**
### Description

- hide repost/fave buttons on track page
- fix slow access check / temporary incorrect access state
- update premium track page copy
- update premium track page and tile styles to match new design
- show both artist pick and premium track tile label where relevant

there are some more fixes to come, but non-blocking and need design

### Dragons

none

### How Has This Been Tested?

local dapp v stage

### How will this change be monitored?

n/a

### Feature Flags ###

existing feature flags

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-pc-qa

File: packages/web/src/common/store/premiumContent/sagas.ts [hunk 1]
```diff
@@ -250,47 +255,64 @@ function* updateCollectibleGatedTrackAccess(
...
-            metadata: {
-              // todo: remove below ts ignore
-              // @ts-ignore
-              premium_content_signature: premiumContentSignatureMap[id]
-            }
-          }
-        })
-      )
-    )
+    // we previously also updated the track's metadata in the store
```
Comment (dharit-tan) [inline packages/web/src/common/store/premiumContent/sagas.ts:312]: Just double-checking here that it's ok to not cache the signatures?

**Review by dharit-tan — APPROVED** [review]

File: packages/web/src/common/store/premiumContent/sagas.ts (hunk 1 above)
Comment (sddioulde) [reply packages/web/src/common/store/premiumContent/sagas.ts:312]: yup thx

File: packages/web/package-lock.json [hunk 2]
```diff
@@ -7270,7 +7270,7 @@
       "resolved": "https://registry.npmjs.org/@types/hoist-non-react-statics/-/hoist-non-react-statics-3.3.1.tgz",
       "integrity": "sha512-iMIqiko6ooLrTh1joXodJK5X9xeEALT1kM5G3ZLhD3hszxBdIEd5C75U834D9mLcINgD4OyZf5uQXjkuYydWvA==",
       "requires": {
-        "@types/react": "*",
+        "@types/react": "18.0.26",
```
[the remark this answers is not in the document]
Comment (sddioulde) [reply packages/web/package-lock.json:7273]: good question. will check with client folks

**Pull request merged**